scripts/run_mcmc.py

```diff
@@ -86,29 +83,10 @@ def create_parser() -> argparse.Namespace:
         default=None,
     )
 
-    parser.add_argument(
-        "--init_tree_mcmc_no",
-        required=False,
-        help="Sample number if mcmc sample is provided as initial tree. "
-        "- line number in file from 1",
-        default=None,
-        type=int,
-    )
-
-    parser.add_argument(
-        "--init_TreeNode",
-        required=False,
-        help="Sample number if mcmc sample is provided as initial tree.",
-        default=None,
-        action="store_true",
-    )
-
     parser.add_argument(
         "--config_fp", required=False, help="Config file path", type=str
     )
 
-    parser.add_argument("--config", required=False, help="Raw config in json", type=str)
-
```

Comment:
removing arguments - in general streamlining the inputs and functionality of this script